frontend/src/views/ddahs/index.tsx

```diff
@@ -83,6 +126,45 @@ export function AdminDdahsView(): React.ReactNode {
                 >
                     Approve DDAH
                 </ActionButton>
+                <ActionButton
+                    icon={FaTrash}
+                    onClick={confirmDDAHDeletion}
+                    disabled={selectedDdahIds.length === 0}
+                >
+                    Delete DDAH
+                </ActionButton>
+                <Modal
+                    show={ddahDeletionConfirmationVisible}
+                    onHide={() => {
+                        setDdahDeletionConfirmationVisible(false);
+                    }}
+                >
+                    <Modal.Header closeButton>
+                        <Modal.Title>Deleting Multiple DDAHs</Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        You are deleting all of the following{" "}
+                        {selectedDdahs?.length} DDAHs!
+                        <br />
+                        <br />
+                        <div style={{ whiteSpace: "pre-line" }}>
```

Comment:
`white-space: pre-line;` is used for formatting code. The users of this system are secretaries. They don't read code.